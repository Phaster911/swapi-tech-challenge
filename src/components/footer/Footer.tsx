import styled from "styled-components";
import MaxWidthWrapper from "../shared/max-width-wrapper/MaxWidthWrapper";
import { QUERIES } from "../../style-constants";

const FooterContentWrapper = styled(MaxWidthWrapper)`
  padding: 1rem;
  margin: 0 auto;
  text-align: center;

  @media ${QUERIES.tabletAndUp} {
    padding: 2rem;
  }
`;

export default function Footer() {
  return (
    <footer>
      <FooterContentWrapper>
        <p>Â© 2024 This Website. All Rights Reserved.</p>
      </FooterContentWrapper>
    </footer>
  );
}
