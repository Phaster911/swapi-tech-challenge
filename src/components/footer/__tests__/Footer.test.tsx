import { render } from "@testing-library/react";
import Footer from "../Footer";

describe("<Footer />", () => {
  it("renders the component", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
