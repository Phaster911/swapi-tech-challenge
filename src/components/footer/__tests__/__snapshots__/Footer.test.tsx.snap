// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Footer /> > renders the component 1`] = `
.c0 {
  width: clamp(16rem,95vw,90rem);
}

.c1 {
  padding: 1rem;
  margin: 0 auto;
  text-align: center;
}

@media (min-width: 750px) {
  .c1 {
    padding: 2rem;
  }
}

<div>
  <footer>
    <div
      class="c0 c1"
    >
      <p>
        Â© 2024 This Website. All Rights Reserved.
      </p>
    </div>
  </footer>
</div>
`;
